roles/etcd/tasks/gen_certs.yml

```diff
@@ -0,0 +1,104 @@
+---
+
+- name: Gen_certs | create etcd cert dir
+  file:
+    path={{ etcd_cert_dir }}
+    group={{ etcd_cert_group }}
+    state=directory
+    owner=kube
```

Comment:
this add a 2nd hardocde for the kube, see roles/kubernetes/node/tasks/install.yml which is the 1st one.
Shall we better off parametrize it?